yourplugin.android.ts

```diff
@@ -1,4 +1,4 @@
-import {Common} from './yourplugin.common';
+import {Common} from "./yourplugin.common";
```

Comment:
It's because tslint in VSCode (with default settings) complains: `' should be "`.

So I Googled and found [this by a TS dev](https://github.com/Microsoft/TypeScript/issues/5811#issuecomment-160187924).